canister_ids.json

```diff
@@ -1,6 +1,5 @@
 {
   "internet_identity": {
-    "identity": "rdmx6-jaaaa-aaaaa-aaadq-cai",
-    "mainnet": "rdmx6-jaaaa-aaaaa-aaadq-cai"
+    "ic": "fgte5-ciaaa-aaaad-aaatq-cai"
```

Comment:
To match the environments as defined by inf it might be worth calling this mainnet.  See e.g. the list of environments here:  https://gitlab.com/dfinity-lab/private/ic-stopgap/-/tree/master/boundary_node/env  Mainnet is also the term used in nns-dapp and extensively in the main ic-repo.  I realize it might seem annoying but the consistency will probably pay off.

Reply:
BTW, is there a reason for removing the identity environment?  I would quite like to add nnsdapp as I have to add that every time I deploy the nnsdapp testnet, and it uses a different canister ID.

Reply:
As discussed, kept identity testnet values and added nnsdapp values.